From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: fix undefined variable

When CREATE is not set there is no default, so it is possible to run
into the test(1) statement in line 116 with an undefined CREATE, which
lets test break.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 IDcheck.sh |    2 ++
 1 file changed, 2 insertions(+)

Index: ltp-full-20080229/IDcheck.sh
===================================================================
--- ltp-full-20080229.orig/IDcheck.sh	2008-03-09 13:35:35.000000000 +0100
+++ ltp-full-20080229/IDcheck.sh	2008-03-09 13:35:55.000000000 +0100
@@ -104,6 +104,8 @@
              CREATE=0 
              ;;
      esac
+  else
+     CREATE=0
   fi
 fi
 
